src/v2/pdf_resources/page_font.h

```diff
@@ -891,6 +879,37 @@ namespace pdflib
   }
 
 
+  void pdf_resource<PAGE_FONT>::init_font_matrix()
+  {
+    LOG_S(INFO) << __FUNCTION__;// << "\t" << json_font.dump(2);
+
+    std::vector<std::string> keys_0 = {"/FontMatrix"};
+
+    if(utils::json::has(keys_0, json_font))
+      {
+        //assert(subtype==TYPE_3);
+        auto result = utils::json::get(keys_0, json_font);
+
```

Comment:
Let's check that result has a size of 6 before we continue

Reply:
Sure, I will also add similar checks in `init_font_bbox`